engine/gamesys/src/gamesys/resources/res_fragment_program.cpp

```diff
@@ -20,13 +20,8 @@ namespace dmGameSystem
 {
     static dmResource::Result AcquireResources(dmGraphics::HContext context, dmResource::HFactory factory, const char* filename, dmGraphics::ShaderDesc* ddf, dmGraphics::HVertexProgram* program)
     {
-        dmGraphics::ShaderDesc::Shader* shader = dmGraphics::GetShaderProgram(context, ddf);
-        if (shader == 0x0)
-        {
-            return dmResource::RESULT_FORMAT_ERROR;
-        }
```

Comment:
The graphics adapters dig the shaders out from the shaderdesc by themselves now.